src/BestBooks.js

```diff
@@ -48,19 +83,49 @@ class BestBooks extends React.Component {
           <h3>{book.title}</h3>
           <p>{book.description}</p>
         </Carousel.Caption>
+        <Button onClick={() => this.props.deleteBook(this.props.book.id)}>Delete</Button>
       </Carousel.Item >
     )
     )
 
     return (
       <>
         <h2>My Essential Lifelong Learning &amp; Formation Shelf</h2>
-
+        <>
+          <Container>
+            <Form onSubmit={this.handleBookSubmit}>
+              <Form.Group controlId="title">
+                <Form.Label>
+                  Title
+                </Form.Label>
+                <Form.Control type="text" />
+              </Form.Group>
+              <Form.Group controlId="description">
+                <Form.Label>
+                  Description
+                </Form.Label>
+                <Form.Control type="text" />
+              </Form.Group>
+              <Form.Group controlId="status">
+                <Form.Label>
+                  Status
+                </Form.Label>
+                <Form.Control type="text" />
+              </Form.Group>
+              <Form.Group controlId="email">
+                <Form.Label>
+                  Email
+                </Form.Label>
+                <Form.Control type="text" />
+              </Form.Group>
```

Comment:
- We need to create a `BookFormModal` component that contains the form elements required to collect the user input needed for creating a new book.
-  Reveal this modal when the "Add Book" button is clicked, and hide the modal when it's closed. 
- You will want to start with making the  `Add Book` button component 👍